app/routes.py

```diff
@@ -1 +1,196 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, jsonify, abort, make_response, request
+from app.models.task import Task
+from app.models.goal import Goal
+from app import db
+from datetime import datetime
+import os, requests
+
+
+tasks_bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals_bp", __name__, url_prefix="/goals")
+
+
+def validate_item_by_id(model, id):
+    try:
+        id = int(id)
+    except:
+        abort(make_response({"message":f"{model.__name__} {id} invalid"}, 400))
+
+    item = model.query.get(id)
+
+    return item if item else abort(make_response({"message":f"{model.__name__} {id} not found"}, 404))
+
+
+@tasks_bp.route("", methods=["POST"])
+def create_task():
+    request_body = request.get_json()
+
+    if "title" not in request_body or "description" not in request_body:
+        abort(make_response({"details":"Invalid data"}, 400))
+
+    new_task = Task(title=request_body["title"],
+                    description=request_body["description"])
+    
+    db.session.add(new_task)
+    db.session.commit()
```

Comment:
Great job using `db.session.<method-name>.` Session gives us access to the following:
* `db` is our app’s instance of SQLAlchemy.
* `session` represents our active database connection.
* By referencing db.session we can use SQLAlchemy’s methods to perform tasks like committing a change to a model, storing a new record of a model, and deleting a record.

By referencing `db.session.add()` you are able to use the SQLAlchemy method to store a new record of the Task model. ⭐